glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
Another intermediate solution is to have ``load_plugins`` be called when the app starts up. I should make that load all plugins, but I also need to make it possible to import a plugin without importing all of them. I'll work on this a bit and see what works best.